engines/sword1/mouse.cpp

```diff
@@ -71,12 +71,10 @@ void Mouse::controlPanel(bool on) { // true on entering cpanel, false when leavi
 	if (on) {
 		savedPtrId = _currentPtrId;
 		_mouseOverride = true;
-		setLuggage(0, 0);
```

Comment:
Is this an intended change?

Reply:
Yes, that's correct 🙂 the luggage operation is to be done by calling the logic function `fnSetLuggage` at the beginning of the main menu and not the `setLuggage` operation in this part. Moreover the game _must not restore the luggage_ (which is basically the action cursor), hence the other change just below this one.